packages/webpack/src/index.ts

```diff
@@ -184,8 +184,23 @@ export default function WebpackPlugin<Theme extends object>(
     }
 
     return plugin
-  }).webpack()
+  })
 }
+
+export default function WebpackPlugin<Theme extends object>(
+  configOrPath?: WebpackPluginOptions<Theme> | string,
+  defaults?: UserConfigDefaults,
+) {
+  return unplugin(configOrPath, defaults).webpack()
+}
+
+export function UnoCSSRspackPlugin<Theme extends object>(
```

Comment:
Named export will make the CJS interop hard - Maybe we should expose a new sub module with default export instead.

Reply:
It can be a minor breaking change, I can bump a new minor version to be safe.